test/SlowTests/Issues/RavenDB_13563.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using Lucene.Net.Index;
+using Lucene.Net.Search;
+using Raven.Client.Documents;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+
+namespace FastTests.Issues
+{
+    public class RavenDB_13563 : RavenTestBase
+    {
+        [Fact]
+        public void Testing()
+        {
+            using (var store = GetDocumentStore())
```

Comment:
What are you actually testing here?

Reply:
It's just a handy test for debugging - not going to be committed to a final PR.